static/src/javascripts/projects/common/modules/commercial/creatives/add-tracking-pixel.js

```diff
@@ -0,0 +1,17 @@
+define([
+    'common/utils/template',
+    'text!common/views/commercial/tracking-pixel.html'
+], function (
+    template,
+    trackingPixelStr
+) {
+    var trackingPixelTpl;
+    function addTrackingPixel($adSlot, url) {
+        if (!trackingPixelTpl) {
```

Comment:
It might be shorter - and more idiomatic - to write

    trackingPixelTpl = trackingPixelTpl || template(trackingPixelStr);